bode/bode/models/task_actions.py

```diff
@@ -33,20 +33,20 @@ def is_interchangable_relation(relation):
     task.title = task_data["title"]
     task.description = task_data["description"]
     task.due_date = task_data["due_date"]
-    task.is_done = task_data["is_done"]
+    task.status = task_data["status"]
 
     db.session.commit()
 
-    if task_data["is_done"]:
+    if task_data["status"] == Status.done.name:
```

Comment:
```suggestion
    if task_data["status"] != Status.todo.name:
```
I think we should propagate `indirectly_done` status in graph of tasks.
Example: T1 <-> T2 and T2 <-> T3 If we check T1, T3 should get `indirectly_done` status in my opinion.

Reply:
not sure about that. T3 is not interchangeable with T1 so it shouldn't be indirectly done imo

Reply:
I will discuss this with a client, couse it's not trivial imho
Solve it however you like if you want, with a posiibility of future backlog refinement to fix it 💪

Reply:
For me interchangeable is equivalence relation. I think PO can say something about that. @koszar91

Reply:
Okay, client responded quickly. He requires (and consider it more logical and thus kinda easier) to have interchangability as equivalence. It is transivie.
So in the world of @gregori0's example, checking T1, makes both T2 and T3 indirectly checked.